.github/workflows/start-release.yml

```diff
@@ -109,6 +109,73 @@ jobs:
             Workflow: <${{github.server_url}}/${{github.repository}}/actions/runs/${{github.run_id}}|${{github.workflow}}>" }}
             ]}
 
+  ci:
+    name: Configure OpenShift CI jobs
+    needs: [variables]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out openshift/release
+        uses: actions/checkout@v3
+        with:
+          repository: openshift/release
+          token: ${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}
+      - name: Check remote branch exists
+        id: check-existing
+        run: |
+          if git ls-remote --quiet --exit-code origin "stackrox-release-$RELEASE"; then
+            echo "::set-output name=branch-exists::true"
+            echo "::notice::OpenShift CI configuration branch exists. Assuming CI is ready."
+          else
+            echo "::set-output name=branch-exists::false"
+          fi
+      - name: Initialize mandatory git config
+        if: steps.check-existing.outputs.branch-exists == 'false'
+        run: |
+          git config user.name "${{github.event.sender.login}}"
+          git config user.email noreply@github.com
+      - name: Branch
+        if: steps.check-existing.outputs.branch-exists == 'false'
+        env:
+          RELEASE: "${{needs.variables.outputs.release}}"
+          CFG_DIR: "ci-operator/config/stackrox/stackrox"
+          JOBS_DIR: "ci-operator/jobs/stackrox/stackrox"
+        run: |
+          PREFIX="$CFG_DIR/stackrox-stackrox-release-$RELEASE"
+          git switch --create "stackrox-release-$RELEASE"
+
+          # Duplicate the main config.
+          yq ".promotion.tag=\"$RELEASE\" | .zz_generated_metadata.branch=\"release-$RELEASE\"" \
+            "$CFG_DIR/stackrox-stackrox-master.yaml" \
+            > "${PREFIX}.yaml"
+
+          # Duplicate the opensource images config.
+          yq ".promotion.tag=\"$RELEASE-stackrox-branding\" | .zz_generated_metadata.branch=\"release-$RELEASE\"" \
+            "$CFG_DIR/stackrox-stackrox-master__stackrox_branding.yaml" \
+            > "${PREFIX}__stackrox_branding.yaml"
+
```

Comment:
I think you are going to need to `yq` out all the `tests` that have no `postsubmit: true` line and `cron: x` or openshift CI will fail. But I may be wrong and maybe it is ok with deleting the job files after the `make update`. Have you been able to test with a `0.x` release?

Reply:
Oh, I didn't test it end-to-end, I just refactored the script you documented and observed `make update` doesn't fail. I'll need your assistance in broader testing.

Reply:
The `presubmits` files are still getting generated after cutting out all the 'non-postsumbit' and 'cron' tests with `--target=[images]`. So I'll keep the `rm` step.

Reply:
https://github.com/openshift/release/pull/31722